packages/@uppy/status-bar/src/StatusBarUI.tsx

```diff
@@ -24,8 +27,48 @@ const {
   STATE_COMPLETE,
 } = statusBarStates
 
+export interface StatusBarUIProps<M extends Meta, B extends Body> {
+  newFiles: number
+  allowNewUpload: boolean
+  isUploadInProgress: boolean
+  isAllPaused: boolean
+  resumableUploads: boolean
+  error: any
+  hideUploadButton?: boolean
+  hidePauseResumeButton?: boolean
+  hideCancelButton?: boolean
+  hideRetryButton?: boolean
+  recoveredState: null | State<M, B>
+  uploadState:
+    | typeof STATE_ERROR
+    | typeof STATE_WAITING
+    | typeof STATE_PREPROCESSING
+    | typeof STATE_UPLOADING
+    | typeof STATE_POSTPROCESSING
+    | typeof STATE_COMPLETE
```

Comment:
This way we can change the single source of truth without thinking to update this too.

```suggestion
  uploadState: typeof statusBarStates[keyof typeof statusBarStates]
```